ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -380,8 +400,6 @@ struct MSFListCellView: View, ConfigurableTokenizedControl {
         var cellContent: some View {
             let children: [MSFListCellStateImpl] = state.children
             let hasChildren: Bool = children.count > 0
-            let horizontalCellPadding: CGFloat = tokens.horizontalCellPadding
-            let leadingViewAreaSize: CGFloat = tokens.leadingViewAreaSize
 
             Button(action: {
                 if hasChildren {
```

Comment:
This seems like an unintended consequence of the 2 scenarios:
 - expanding children
 - selecting a cell

https://user-images.githubusercontent.com/68076145/163840211-a8fab788-f765-475a-9ad7-1bba8a33e1b0.mov


I believe this should be addressed in a way that we don't allow selection of parent cells as they're just expanding/collapsing their children.

Thoughts?

Reply:
I can update it for now, but selecting folders with subfolders in Outlook is an example of where exanding / selecting should both be doable.